File: Beecrowd/1094 Experiências/main.py
N = int(input())
somaC = 0 
somaR = 0
somaS = 0

for i in range(N):
  entrada = [str(a) for a in input().split()]
  quantia = int(entrada[0])
  tipo = entrada[1]
  if tipo == 'C':
    somaC += quantia
  elif tipo == 'R':
    somaR += quantia
  elif tipo == 'S':
    somaS += quantia
  
totalCobaia = somaC + somaR + somaS
porC = (float)(100 * somaC) / totalCobaia
porR = (float)(100 * somaR) / totalCobaia
porS = (float)(100 * somaS) / totalCobaia

print(f"Total: {totalCobaia} cobaias")
print(f"Total de coelhos: {somaC}")
print(f"Total de ratos: {somaR}")
print(f"Total de sapos: {somaS}")
print("Percentual de coelhos: {:.2f} %".format(porC))
print("Percentual de ratos: {:.2f} %" .format(porR))
print("Percentual de sapos: {:.2f} %" .format(porS))

File: Beecrowd/1094 Experiências/main.c
#include <stdio.h>
 
int main() {
 
  int i, N, quantia, somaC = 0, somaR = 0, somaS = 0, totalCobaia;
  double porC, porR, porS;
  char tipo;

  scanf("%d", &N);

  for(i = 0; i < N; i++){
    do{
      scanf("%d %c", &quantia, &tipo);
    }while(tipo != 'C' && tipo != 'R' && tipo != 'S');
    switch (tipo){
      case 'C':
        somaC += quantia;
      break;
      case 'R':
        somaR += quantia;
      break;
      case 'S':
        somaS += quantia;
    }
  }
  
  totalCobaia = somaC + somaR + somaS;
  porC = (double)(100 * somaC) / totalCobaia;
  porR = (double)(100 * somaR) / totalCobaia;
  porS = (double)(100 * somaS) / totalCobaia;

  printf("Total: %d cobaias\n", totalCobaia);
  printf("Total de coelhos: %d\n", somaC);
  printf("Total de ratos: %d\n", somaR);
  printf("Total de sapos: %d\n", somaS);
  printf("Percentual de coelhos: %.2lf %%\n", porC);
  printf("Percentual de ratos: %.2lf %%\n", porR);
  printf("Percentual de sapos: %.2lf %%\n", porS);
 
    return 0;
}